docs/source/introduction.md

```diff
@@ -9,10 +9,10 @@
 - [Agda programming language](https://github.com/agda/agda)
   - version >= 2.6.4 && < 2.6.5
 - [Agda standard library](https://github.com/agda/agda-stdlib)
-- Agda library `agda2hs`
+- Agda library `base`
   - this Agda library is included in the `agda2hs` repository; see
-    [`agda2hs.agda-lib`](https://github.com/agda/agda2hs/blob/master/lib/agda2hs/agda2hs.agda-lib)
-  - you can navigate the library in [HTML format](https://agda.github.io/agda2hs/lib/agda2hs/),
+    [`base.agda-lib`](https://github.com/agda/agda2hs/blob/master/lib/base/base.agda-lib)
+  - you can navigate the library in [HTML format](https://agda.github.io/agda2hs/lib/base/),
```

Comment:
Reminder to check that these links still work and the documentation generating scripts do their job.